yaksh/templates/yaksh/quizzes_user.html

```diff
@@ -1,6 +1,21 @@
 {% extends "user.html" %}
 {% block pagetitle %} {{ title }} {% endblock %}
 {% block main %}
+<div class="row well">
+    <form action="{{ URL_ROOT }}/exam/quizzes/" method="post" id="custom-search-form" class="form-search form-horizontal pull-right">
+        {% csrf_token %}
+        <div class="col-md-12">
+            <div class="input-group">
+                <span class="input-group-addon" id="basic-addon1">Search Course</span>
+                <input type="text" name="course_code" class="form-control" placeholder="Course Code">
+                <span class="input-group-btn">
+                    <button class="btn btn-default" type="submit">Search</button>
+                    <button class="btn btn-default" type="button" name="button" onClick='location.replace("{{URL_ROOT}}/exam/quizzes/");'>Cancel</button>
```

Comment:
The cancel button refreshes the page because after the search is executed only hidden courses are displayed. Hitting cancel will reload and display the regular, open, non-code courses